src/libs/ajax.js

```diff
@@ -495,8 +495,8 @@ const Buckets = signal => ({
 
       getObject,
 
-      rename: async newName => {
-        await copy(newName)
+      rename: async (newName, bucket) => {
+        await copy(newName, bucket)
```

Comment:
The difference between copy and rename is that rename presumes the bucket, so this should use `bucket` as defined above, and remain a function of 1 argument.

Thanks for catching this!

Reply:
Ah, got it, makes sense. Thanks!